### DISCUSSION TITLE
Is any type of data redundancy compatible with this?

### DISCUSSION THREAD

**Discussion by Zerophase** [Chia-Network/chia-blockchain#5144] [Support]
I will be plotting on NVME drives in RAID 0. Currently, figuring out the max a board can support. I might be limited to between four and ten drives. But, even with four 16tb HDDs get filled up fairly fast. Is there any type of data redundancy that still allows for hotswapping when a drive is full, in this use case? Is deduplication for long term storage possible?

Ideally, I'm looking for something where I can pull a disk when full, and not have to worry about data loss from the shear amount of disks that will be in the farming rig / rigs.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
